docs/bond.md

```
@@ -0,0 +1,85 @@
+# 美国国债收益率数据
+
+## 数据来源
+
+[美国财政部官方网站](https://home.treasury.gov/resource-center/data-chart-center/interest-rates/TextView?type=daily_treasury_yield_curve&field_tdr_date_value=2026)
+
+XML API 接口格式：
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)